cli.py

```diff
@@ -1,10 +1,36 @@
 #!/usr/bin/env python
+# coding=latin-1
 
-import cmd, getpass, json, os, pprint, re, sys, time
+import cmd, getpass, json, os, pprint, re, sys, time, random
 import requests, execjs
 from lxml import html
 from bs4 import BeautifulSoup
 
+class ArtDeco():
+    bunnies = [
+        """  (\(\\\n (='.')\no(__")")""",
+        """(\__/)\n(='.'=)\n(")_(")""",
+        """ (\_/)\n=(^.^)=\n(")_(")""",
+        """(\ /)\n( . .)\nc(")(")""",
+        """(\__/)\n(>'.'<)\n(")_(")""",
+        """::: (\_(\\\n *: (=' :') :*\n•.. (,(”)(”)¤°.¸¸.•´¯`»""",
+        """   __//   \n  /.__.\  oops\n  \ \/ /  \n__/    \  \n\-      ) \n \_____/  \n___|_|____\n   " " """,
+        """|￣￣￣￣￣| \n|  LITTLE  |\n|  BUNNY   |\n|  LOVES   |\n|   YOU    |\n|    -xt2  |\n|＿＿＿＿＿| \n(\_/)  || \n(•ㅅ•) || \n/ 　 づ """,
+        """...........(\_/)\n...........( '_')........................ ☻...HELP!!!!\n...../++++++++++++++\======¦¦¦D -------- /▇ \\ \n/====================\..................  ||\n\_@___@___@___@___@__/"""
+    ]
+    def __init__( self ):
+        self.bunnyOTD = random.choice(self.bunnies)
```

Comment:
bunnyOTD -> bunny.

Also, please put extra spaces in parentheses and brackets.